static/styles/subscriptions.css

```diff
@@ -1070,13 +1040,19 @@ h4.stream_setting_subsection_title {
     }
 }
 
-.stream-creation-body input[type="radio"] {
-    margin: 3.5px 0 1px;
-}
```

Comment:
Removed this CSS class as there do not exists any input[type="radio"] in `stream-creation-body`